custom_model_runner/datarobot_drum/runtime_parameters/runtime_parameters.py

```diff
@@ -48,7 +48,7 @@ class RuntimeParameters:
     PARAM_PREFIX = "MLOPS_RUNTIME_PARAM"
 
     @classmethod
-    def get(cls, key):
+    def get(cls, key, default=None):
```

Comment:
@elatt, 
I want to allow default values instead of just raising the `ValueError`. What do you think?

Reply:
No, I object to this. We intentionally don't allow _defaults_ because it indicates an issue with the runtime param framework if the env var doesn't exist. The framework always provides a default of `None`. The _hack_ we are doing with envs is really just that, it's a hack and that is why it feels necessary to have this. The proper solution would be for envs to be able to properly define the runtime params and in that case they will all have defaults (inherited from the definitions).